src/main/java/com/rarchives/ripme/ripper/rippers/EightmusesRipper.java

```diff
@@ -122,14 +123,10 @@ else if (!subUrl.startsWith("http://")) {
                     }
                     try {
                         logger.info("Retrieving full-size image location from " + parentHref);
-                        Thread.sleep(1000);
                         image = getFullSizeImage(parentHref);
                     } catch (IOException e) {
                         logger.error("Failed to get full-size image from " + parentHref);
                         continue;
-                    } catch (InterruptedException e) {
```

Comment:
@metaprime This seems to be a maven issue. Trying to build my old fork (Which built and ran fine a week ago) now fails with 

```[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:compile (default-compile) on project ripme: Compilation failure
[ERROR] /home/USER/ripme/src/main/java/com/rarchives/ripme/ripper/rippers/EightmusesRipper.java:[130,23] exception java.lang.InterruptedException is never thrown in body of corresponding try statement```